pkg/cloudprovider/providers/vsphere/nodemanager.go

```diff
@@ -22,10 +22,13 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/vmware/govmomi/object"
 	"k8s.io/api/core/v1"
 	k8stypes "k8s.io/apimachinery/pkg/types"
+	cloudprovider "k8s.io/cloud-provider"
 	"k8s.io/klog"
 	"k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere/vclib"
+	kubeletapis "k8s.io/kubernetes/pkg/kubelet/apis"
```

Comment:
This package no longer holds the zone label constants, it was moved to `k8s.io/api/core/v1`. You may have to rebase to see those changes :)

Reply:
Thanks for pointing out :) Rebased and fixed the code as per the new changes.